aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -864,14 +864,15 @@ class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknownHtmlT
     }
 
     private fun loadImages() {
+        val maxDimension = maxImagesWidth
```

Comment:
I think we can remove `val maxDimension` and use `maxImagesWidth` directly in this method